users/tests/test_models.py

```diff
@@ -53,3 +53,15 @@ def test_needs_vote_affirmation(self):
         )
 
         self.assertTrue(member3.needs_vote_affirmation)
+
+class UserGroupsModelsTestCase(TestCase):
+    def test_create_usergroup(self):
+        group = UserGroup.objects.create(
+            name='PLUG',
+            url='http://meetup.com/plug',
+            url_type=0
```

Comment:
Style nit: We can use `UserGroup.TYPE_MEETUP` instead of `0` if we follow my advice above.

Reply:
Done.